.github/workflows/issue_labeler.yml

```
@@ -0,0 +1,35 @@
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    permissions:
+      issues: write
+
+    strategy:
+      matrix:
+        template: [bug_report.yml]
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)